ui/src/store/auth.test.ts

```diff
@@ -60,4 +60,58 @@ describe('Store functions', () => {
       }
     );
   });
+
+  it('can get the rating for a resource', (done) => {
```

Comment:
Can we change it to get `user's rating for a resource` instead of rating for a resource